charts/openstack-cloud-controller-manager/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: v1.27.1
+appVersion: v1.27.2
 description: Openstack Cloud Controller Manager Helm Chart
 icon: https://object-storage-ca-ymq-1.vexxhost.net/swift/v1/6e4619c416ff4bd19e1c087f27a43eea/www-images-prod/openstack-logo/OpenStack-Logo-Vertical.png
 home: https://github.com/kubernetes/cloud-provider-openstack
 name: openstack-cloud-controller-manager
-version: 2.27.2
+version: 2.27.3
```

Comment:
What should I do with this now? @zetaab?

Reply:
always increasing the version is the way to go